graphene_linked_events/tests/snapshots/snap_test_api.py

```diff
@@ -739,6 +740,7 @@
                     "offers": [{"isFree": False}],
                     "pEvent": {
                         "autoAcceptance": True,
+                        "autoAcceptanceMessage": None,
```

Comment:
- [ ] autoAcceptanceMessage should not be `None`

Reply:
- [ ] IT should not be `{fi:null}` either :)

Reply:
Finally ended up to use the same style as occurrences schema uses with study levels and venues. In my opinion it's not as good solution as it would be to use the translations in "map format", which is the way how the LinkedEvents returns the translations, but at least it is easier to maintain when more supported languages are added and all the same infromation is still available - it's just in a list format.